lambda/lambda.ml

```diff
@@ -1565,6 +1620,7 @@ let block_of_module_representation ~loc = function
              to after slambdaeval (or maybe during?). *)
           | Splice_variable _ ->
             Misc.fatal_error "Layout poly doesn't support mixed modules yet")
+            (* Slambda.raise ~loc (Unsupported "mixed modules")) *)
```

Comment:
I really wanted to use this but it causes a dependency loop between slambda and lambda, do you have any suggestions or should I just leave it as a fatal error on the assumption we'll fix this soon.